packages/devtools/src/publishing.ts

```diff
@@ -11,20 +11,12 @@ import { promiseMap } from './utils'
 const PUBLISH_OPTS = { anchor: false, publish: false }
```

Comment:
@oed IIRC we needed to set these to create from genesis and apply commits, but I'm wondering why we needed to set `publish: false` here?
I'd expect when we deploy a datamodel to a different network, we want it to be propagated to the network, not just the node we write to, no?
Is it a problem if the same genesis and signed commits are published multiple times please?
cc @stbrody

Reply:
Is this always used with `deterministic:true`?  Or is this sometimes used to create unique documents as well?  Certainly for unique documents I see no reason not to publish anytime one is created.

For deterministic documents, I do think that if the same document is "created" multiple times, you will publish the genesis commit each time, so that might be what we were trying to avoid here?  It wouldn't be a "problem" per-se if the same genesis commit were published multiple times, but it does create extra unnecessary pubsub traffic, which will affect how well the network scales.

Reply:
also not sure why we wouldn't want `anchor` to be true every time?

Reply:
@stbrody the problem that might occur (I'm not sure, we should investigate) is that if you call with `anchor: true` then we will request an anchor on genesis even if the stream already has updates. I guess it shouldn't matter that much but it is a non-needed code path in this case.

@PaulLeCam I think technically we don't **need** to set any of these. It's more of an optimization iirc.

Reply:
We should double-check, but I think in the case of deterministic tile documents this will be fine, as we do try to sync the stream by default so I'm *fairly* sure we'd be requesting an anchor for the newest commit of the stream, not the genesis commit.